packages/@sanity/client/README.md

```diff
@@ -32,14 +32,22 @@ const client = sanityClient({
 
 Initializes a new Sanity Client. Required options are `projectId` and `dataset`.
 
-### Fetch a single document
+### Fetch an entire single document
```

Comment:
Why "entire"? What would be the alternative?

Reply:
The alternative would be a _partial_ document. Maybe "complete", "full" is better? Or just leave it as is?